### ISSUE TITLE
Simplify advice and obligation assignment

### ISSUE THREAD

**Issue by andrewclymer** [davidjbrossard/alfa-authorization-language#8]
The following obligation assigns the Action attribute to the obligation with the same value as the Action attribute has in the current context.

```
   on permit
        {
            obligation Audit
            {
                When = CurrentDateTime
                Who = Subject.Email
                Description = "Accessed the Purchase Order system"
                Action = Action
            }
        }
```

Suggest simplifying to

```
   on permit
        {
            obligation Audit
            {
                When = CurrentDateTime
                Who = Subject.Email
                Description = "Accessed the Purchase Order system"
                Action 
            }
        }
```

**Comment by davidjbrossard** [comment]
Agreed. The annoying thing with advice/obligations is to have to specify the name of the placeholder even when the content itself could be used as the name of the placeholder (Action = Action vs Action).

**Comment by TheoDimitrakos** [comment]
This is still a bit unclear to me.   What is the syntax of the obligation in this situation and why are  attributes for When, Who, Description and Action are treated  differently. Do you assume that Obligation  Audit is defined to have aonly one Action and this Action has value Action?  What if there were multiple possible asction values assocaited with an oblgiation?  

Would it be 

  on permit
        {
            obligation Audit
            {
                When = CurrentDateTime
                Who = Subject.Email
                Description = "Accessed the Purchase Order system"
                Action1  
            }
        }

and 

  on permit
        {
            obligation Audit
            {
                When = CurrentDateTime
                Who = Subject.Email
                Description = "Accessed the Purchase Order system"
                Action2  
            }
        }

or 

(In general I agree with the principle  of using content itself  as a placeholder where possible but it is not clear to me why and how this can simpify obligations specifically. )
